world_population_analysis.yaml

```diff
@@ -0,0 +1,18 @@
+stages:
+  - name: worldpopulation
+    scheduler:
+      scheduler_type: 'singlestep-stage'
+      parameters:
+        outputdir: '{workdir}'
+        outputfile: '{workdir}/world_population_analysis.html'
+      step:
+        process:
+          process_type: 'string-interpolated-cmd'
+          cmd: 'jupyter nbconvert --output-dir="{outputdir}" world_population_analysis.ipynb'
```

Comment:
In theory we could remove `jupyter nbconvert` since it is set as `ENTRYPOINT` in the `Dockerfile`. On the other hand, it will be weird the `--output-dir="{outputdir}"` parameter..